blog/content/post/2019-04-18-USB-sniffing-with-OpenVizsla.md

```diff
@@ -0,0 +1,195 @@
+---
+title: USB Sniffing With OpenVizsla
+cover: /covers/sniffing_usb_cover.jpg
+author: lukasz.wcislo
+layout: post
+published: false
+date: 2019-04-18
+
+tags:
+  - USB
+  - USB sniffing
+  - OpenVizsla
+categories:
+  - Miscellaneous
+  - Security
+
+---
+
+### Background
+
+**"OpenVizsla is a project to design a device that will allow the capture and
+inspection of USB traffic that will help with the reverse engineering and
+debugging of proprietary USB devices, and will also be an invaluable tool for
+developers working with USB and especially those who are using USB in
+embedded designs."**
```

Comment:
@lwcislo shall we treat it as abstract? @artur-rs header is not aligned with most recent template?

Reply:
@pietrushnic it is not merged yet - making changes and I will address them to @lwcislo